tests/DetectsChangesTest.php

```diff
@@ -299,6 +299,7 @@ public function it_will_store_the_values_when_deleting_the_model()
         $expectedChanges = collect([
             'attributes' => [
                 'name' => 'my name',
+                'text' => null,
```

Comment:
These are needed now cause `getModelAttributeValue` returns at least `null` for every requested attribute. I think that this is also a better behaviour cause with this all wanted attributes exist and it's also default eloquent and database behaviour to set `null|empty` as value if nothing is set.
If you think that this will/could break something we have to find a way around. My first idea would be an `throw new Exception` and a `try-catch` in the `logChanges` method.